src/accountingservice/Dockerfile

```
@@ -26,6 +26,7 @@ COPY --from=publish /app/publish .
 USER root
 RUN mkdir -p "/var/log/opentelemetry/dotnet"
 RUN chown app "/var/log/opentelemetry/dotnet"
+RUN chmod 755 "/app/instrument.sh"
```

```suggestion
RUN chown app "/app/instrument.sh"
```

The `app` user seems to be the one running the script, would it work just setting it as the owner? (no execution permissions to others)

---

Let me check if this works and come back :)

---

It works. Commited

---

Thanks for the double check!